metadata-ingestion/src/datahub/ingestion/source/sql/sql_common.py

```diff
@@ -572,14 +572,14 @@ def gen_schema_key(self, db_name: str, schema: str) -> PlatformKey:
             database=db_name,
             schema=schema,
             platform=self.platform,
-            instance=self.config.env,
+            instance=self.config.platform_instance,
```

Comment:
we'll handle this change in a separate PR -> draft PR is here (https://github.com/linkedin/datahub/pull/4279). 
Once that is merged, you shouldn't need to fix these files.

Reply:
It was merged